docs/user/generated_config.adoc

```diff
@@ -455,6 +455,12 @@ Enable support for procedural macros, implies `#rust-analyzer.cargo.runBuildScri
 Internal config, path to proc-macro server executable (typically,
 this is rust-analyzer itself, but we override this in tests).
 --
+[[rust-analyzer.procMacro.dummies]]rust-analyzer.procMacro.dummies (default: `{}`)::
```

Comment:
“ignored” might be a better name — while we implement this as an identity macro, from the user’s pov it is ignored. Similarly, it’s better to write help from user’s pov: “identity expander” is jargony.